docs/banned_addons.md

```diff
@@ -1,8 +1,37 @@
+# banned\_addons
+
 [<-Back-to:Characters](database-characters.md)
 
-Column | Type | Description
---- | --- | ---
-Id | INT UNSIGNED | 
-Name | VARCHAR(255) | 
-Version | VARCHAR(255) | 
-TIMESTAMP | TIMESTAMP | 
+**The \`banned\_addons\` table**
+
+**Structure**
+
+| Field          | Type         | Attributes | Key | Null | Default           | Extra  | Comment |
+| -------------- | ------------ | ---------- | --- | ---- | ----------------- | ------ | ------- |
+| [Id][1]        | INT          | UNSIGNED   | PRI | NO   |                   |        |         |
+| [Name][2]      | VARCHAR(255) |            |     | NO   |                   | UNIQUE |         |
+| [Version][3]   | VARCHAR(255) |            |     | NO   | ''                | UNIQUE |         |
+| [Timestamp][4] | TIMESTAMP    |            |     | NO   | CURRENT_TIMESTAMP |        |         |
+
+[1]: #Id
+[2]: #Name
+[3]: #Version
+[4]: #Timestamp
```

Comment:
write these in small letters otherwise it doesn't work. Only need to change these lines